Fedora/hpx.spec

```diff
@@ -177,22 +228,29 @@ done
 . /etc/profile.d/modules.sh
 for mpi in openmpi mpich '' ; do
   test -n "${mpi}" && module load mpi/${mpi}-%{_arch} && mkdir -p %{buildroot}/${MPI_BIN}
-  %make_install -C ${mpi:-serial}
-  sed -i '1s@env python@python2@' %{buildroot}/%{_bindir}/{hpx*.py,hpxcxx}  %{buildroot}%{_libdir}/cmake/HPX/templates/hpx{cxx,run.py}.in
-  chmod +x  %{buildroot}%{_libdir}/cmake/HPX/templates/hpx{cxx,run.py}.in
+  pushd ${s:-.}/${mpi:-serial}
+  %make_install
+  if [ -n "$mpi" ]; then
+  libdir=%{_libdir}/${mpi}/lib
+  else
+  libdir=%{_libdir}
+  fi
+  sed -i '1s@env python@python2@' %{buildroot}/%{_bindir}/{hpx*.py,hpxcxx}  %{buildroot}${libdir}/cmake/HPX/templates/hpx{cxx,run.py}.in
```

Comment:
You could just use `${MPI_LIB:-%{_libdir}}` instead of `${libdir}` here and drop the `if` block above.

Reply:
Ok, good to know